tests/tests-core/src/main/java/com/gentics/mesh/core/language/AtomicLanguageTest.java

```diff
@@ -16,15 +15,19 @@ public class AtomicLanguageTest extends AbstractMeshTest {
 	@Test
 	public void testLanguageIndex() {
 		try (Tx tx = tx()) {
-			LanguageDao languageRoot = boot().languageDao();
-			try (Tx tx2 = tx()) {
-				assertNotNull(languageRoot);
-				HibLanguage lang = languageRoot.create("Deutsch1", "de1");
-				lang = languageRoot.create("English1", "en1");
-				tx2.success();
-			}
-
-			assertNotNull(languageRoot.findByLanguageTag("en1"));
+			LanguageDao languageRoot = tx.languageDao();
```

Comment:
Since we have a complex constraint on a language entity, only one language per tx can be created.